### PR TITLE
[#9144] Update user map for Aug 2018

### LINKED ISSUE #9144: Update user map for Aug 2018
To be done after #8943

Entries to consider

Country| Institute
-- | --
Australia | Brighton Secondary School
Australia | Killara High School
Australia | RMIT
Australia | University of Queensland
Australia | University of Sydney
Belgium | Artevelde University College
Belgium | GTI Mortsel
Belgium | KU Leuven
Brazil | AEDB
Brazil | UNIFEV - Centro Universitário de Votuporanga - SP
Brazil | Universidade Católica de Brasília
Canada | Bow Valley College
Canada | Conestoga College
Canada | Dalhousie University
Canada | Humber College
Canada | Okanagan College
Canada | Simon Fraser University
Canada | University of Lethbridge
Canada | University of Manitoba
Chile | Pontificia Universidad Católica de Valparaíso
Equador | UEES University
France | Grenoble Ecole de Management
Hong Kong | HKUST
Hong Kong | The University of Hong Kong
India | Institute of Technology, Gopeshwar
India | Symbiosis International University
India | VALTECH
Japan | Ritsumeikan Asia Pacific University
Kazakhstan | Method Coding School
Kenya | ALX
Malaysia | KDU Penang UC Sdn Bhd
Malaysia | Sunway College
Malaysia | Taylor’s University
Mauritius | African Leadership University
Mexico | UNID
Mexico | Universidad Panamericana
Nederland | MHC Dommel
Netherlands | Hanze University of Applied Sciences
Netherlands | Innovation Lab
New Zealand | Massey University
New Zealand | University of Auckland
New Zealand | Victoria University of Wellington
Panama | Universidad Tecnológica de Panamá
Philippines | ATENEO DE NAGA UNIVERSITY
Philippines | De La Salle Lipa
Philippines | De La Salle Santiago Zobel School
Rwanda | UCA
Singapore | Greenwood Primary School
Singapore | Institue of Technical Education
Singapore | Nanyang Technological University
South Korea | Handong Global University
Sri Lanka | University of Kelaniya
Sweden | Stockholm University
Switzerland | SHMS
Taiwan | Kainan University
UAE | Murdoch University
UK | Aston University
UK | Imperial College London
UK | London School of Economics
UK | Newcastle University
UK | Nord Anglia Education
UK | The University of Sheffield
UK | University of Manchester
UK | University of Portsmouth
USA | Amherst College
USA | Amherst College
USA | Baylor University
USA | Beaufort County Early College High School
USA | Becker College
USA | Belmont University
USA | Bennington College
USA | Bentley University
USA | Cal Poly Pomona
USA | California State University, Sacramento
USA | Cisco College
USA | Colorado School of Mines
USA | Creighton University
USA | Doherty High School
USA | Drexel University
USA | Florida International University
USA | Frontier Nursing University
USA | Garrett-Evangelical Theological Seminary
USA | Gaston College
USA | Grand Canyon University
USA | Grand Valley State University
USA | Grand Valley State University
USA | Hult International Business School
USA | Indiana University
USA | Intensive English Institute
USA | Iowa State University
USA | Johns Hopkins University
USA | Kennesaw State University
USA | Las Positas College
USA | Loyola Marymount University
USA | Mountain View High School
USA | New Mexico State University
USA | North Dakota State University
USA | North Seattle College
USA | Northeastern University
USA | Northern Kentucky University
USA | Ohio State University
USA | Ohio State University
USA | Plymouth State University
USA | Rochester Institute of Technology
USA | Saint Louis University
USA | San Jose State University
USA | Tamalpais High School
USA | Texas A&M University
USA | The College of New Jersey
USA | U.S. Army
USA | University at Albany, SUNY
USA | University of Arkansas
USA | University of Colorado Boulder
USA | University of Colorado, Colorado Springs
USA | University of Florida
USA | University of Florida
USA | University of Maryland
USA | University of Nevada, Reno
USA | University of New Mexico
USA | University of North Carolina at Charlotte
USA | University of North Carolina at Wilmington
USA | University of North Georgia
USA | University of South Alabama
USA | University of South Dakota
USA | University of Southern California
USA | University of Texas at El Paso
USA | University of Texas at El Paso
USA | University of Texas El Paso
USA | University of Texas, Austin
USA | Utah State University
USA | Virginia Commonwealth University
USA | Washington State University
USA | Wellesley College
USA | Western Washington University
USA | Whatcom Community College
USA | Wheaton College
USA | William Mulholland Middle School

### PR DESCRIPTION
**Pull request by Ayush517**
Fixes #9144 

Updated the userMapData.json file for August 2018.

### CODE REVIEW

**Comment by RonakLakhotia** [comment]
Have you run `sort-usermap.js` to sort the entry?

**Comment by Ayush517** [comment]
> Have you run `sort-usermap.js` to sort the entry?

No. I've done by manually.
Checked all the entries twice.

File: src/web/data/userMapData.json [hunk 1]
```diff
@@ -610,6 +622,7 @@
       "NIS Almaty Ph.M"
     ],
     "Kenya": [
+      "ALX",
```
Comment (RonakLakhotia) [inline src/web/data/userMapData.json]: Is there a full form for the abbreviation?

File: src/web/data/userMapData.json [hunk 2]
```diff
@@ -890,6 +916,7 @@
       "Tomsk State University"
     ],
     "Rwanda": [
+      "UCA",
```
Comment (RonakLakhotia) [inline src/web/data/userMapData.json]: similarly here

File: src/web/data/userMapData.json [hunk 3]
```diff
@@ -1047,12 +1076,14 @@
       "La Cote International School",
       "Medtronic",
       "Pädagogische Hochschule St. Gallen",
+      "SHMS",
```
Comment (RonakLakhotia) [inline src/web/data/userMapData.json]: and here

Comment (Ayush517) [reply src/web/data/userMapData.json]: The issue mentioned just the abbreviations in these 3 cases.

Comment (Ayush517) [reply src/web/data/userMapData.json]: There are many abbreviations like these in the current file, so I thought that just the exact name mentioned in the issue has to be added to the list.

Comment (RonakLakhotia) [reply src/web/data/userMapData.json]: It is better to use full forms. You can search for their full forms. Follow #9190

Comment (Ayush517) [reply src/web/data/userMapData.json]: I can do that if but can we make it a new d.FirstTimers issue to update all abbreviations as Full Forms? That might be more useful as it the new issue will require changing all the abbreviations in the file and not just these 3.

Comment (wkurniawan07) [reply src/web/data/userMapData.json]: @Ayush517, @RonakLakhotia was asking you to use full forms only for entries you added. Existing entries are not the concern of this PR.

> > Have you run `sort-usermap.js` to sort the entry?
> 
> No. I've done by manually.
> Checked all the entries twice.

You still need to run that script for two reasons: (1) the sorting logic may be different from what you had in mind, and (2) the sorting script is immune to human errors.

Comment (Ayush517) [reply src/web/data/userMapData.json]: 1.) I'll update the abbreviations for UCA and SHMS, there's no full form for ALX
2.) I'll run the script on the changed file.
I'll have to close this pull request?

Comment (RonakLakhotia) [reply src/web/data/userMapData.json]: No need to close this pull request, update the changes in your local branch and push them

**Comment by Ayush517** [comment]
@RonakLakhotia @wkurniawan07  Is it right this time?

**Comment by Ayush517** [comment]
@wenmogu can you please review this and let me know if anything needs to be updated.

File: src/web/data/userMapData.json [hunk 4]
```diff
@@ -50,7 +52,7 @@
       "Narangba Valley SHS",
       "Photography Studies College",
       "Queensland University of Technology",
-      "RMIT University",
+      "RMIT University (RMIT)",
```
Comment (wenmogu) [inline src/web/data/userMapData.json]: Royal Melbourne Institute of Technology (RMIT)

File: src/web/data/userMapData.json [hunk 5]
```diff
@@ -148,6 +151,7 @@
       "University of Botswana"
     ],
     "Brazil": [
+      "AEDB",
```
Comment (wenmogu) [inline src/web/data/userMapData.json]: full form pls

File: src/web/data/userMapData.json [hunk 6]
```diff
@@ -360,7 +367,8 @@
       "VIA University College"
     ],
     "Ecuador": [
-      "Escuela Superior Politecnica Del Litoral"
+      "Escuela Superior Politecnica Del Litoral",
+      "UEES University"
```
Comment (wenmogu) [inline src/web/data/userMapData.json]: full form pls

File: src/web/data/userMapData.json [hunk 7]
```diff
@@ -531,10 +540,12 @@
       "Pearl Academy",
       "Rajalakshmi School of Business",
       "Swami Ram Ahimalayan University",
+      "Symbiosis International University",
       "Tech Mahindra",
       "Thapar Institute of Engineering & Technology",
       "Thiagarajar College of Engineering",
       "USICT, New Delhi",
+      "VALTECH",
```
Comment (wenmogu) [inline src/web/data/userMapData.json]: Valtech

File: src/web/data/userMapData.json [hunk 8]
```diff
@@ -702,6 +718,9 @@
     "Morocco": [
       "George Washington Academy"
     ],
+    "Nederland": [
```
Comment (wenmogu) [inline src/web/data/userMapData.json]: this is netherlands if i m not wrong?

File: src/web/data/userMapData.json [hunk 9]
```diff
@@ -825,11 +848,14 @@
       "aGreatLife",
       "Asian Institute of Management",
       "Ateneo de Manila University",
+      "ATENEO DE NAGA UNIVERSITY",
```
Comment (wenmogu) [inline src/web/data/userMapData.json]: pls dont capitalize all characters

File: src/web/data/userMapData.json [hunk 10]
```diff
@@ -1278,6 +1316,7 @@
       "BYU-Idaho",
       "C/3-13IN Basic Training",
       "Cabrini University",
+      "Cal Poly Pomona",
```
Comment (wenmogu) [inline src/web/data/userMapData.json]: duplicate entry

File: src/web/data/userMapData.json [hunk 11]
```diff
@@ -1930,6 +1979,7 @@
       "Tribeca Flashpoint Academy",
       "Truman State University",
       "Tufts University",
+      "U.S. Army",
```
Comment (wenmogu) [inline src/web/data/userMapData.json]: full form pls

File: src/web/data/userMapData.json [hunk 12]
```diff
@@ -2011,14 +2061,14 @@
       "University of Nebraska",
       "University of Nebraska at Kearney",
       "University of Nebraska-Lincoln",
-      "University of Nevada",
+      "University of Nevada, Reno",
```
Comment (wenmogu) [inline src/web/data/userMapData.json]: y the deletion?

Comment (Ayush517) [reply src/web/data/userMapData.json]: had to add Reno to it

Comment (wenmogu) [reply src/web/data/userMapData.json]: but University of Nevada has many branches, please check the relevant issue for the previous entry and make sure that the deletion is appropriate.

Comment (Ayush517) [reply src/web/data/userMapData.json]: Oh, I just saw it. Will revert this change back as it was

**Comment by Ayush517** [comment]
Can I close this PR? I don't know what happened during the last merge but I think there's something happening which is not supposed to happen.

**Comment by amrut-prabhu** [comment]
@Ayush517 Are you referring to the changes in the [Java files](https://github.com/TEAMMATES/teammates/pull/9478/commits/4b63a947ef1d365d715da034c8195370242ee7db)?
You can try hard resetting back to your commit before the weird stuff started happening, and then force push?

**Comment by Ayush517** [comment]
yes the java files
Instead of going through all the problems of hard reset and multiple commits to this PR, wouldn't it be good if I open another one with just one commit?

**Pull request closed without merging**